packages/react/src/evm/components/ConnectWallet/index.tsx

```diff
@@ -248,7 +250,7 @@ export const ConnectWallet: React.FC<ConnectWalletProps> = ({
               </span>
             )
           ) : (
-            <span style={{ whiteSpace: "nowrap" }}>Connect Wallet</span>
+            <span style={{ whiteSpace: "nowrap" }}>{btnTitle}</span>
```

Comment:
why not just pass `children` at this point?

Reply:
It is a dropdown so string was the first got in mind. But I think `children` will be better in case someone want to pass a styles text `<span style={styles}></span>`
I will implement this change, Thank you🙏🏻

Reply:
updated check again, please 🙏